docs/reference/toolbox-reference.md

```diff
@@ -289,6 +289,25 @@ total 8
 
 In the previous example, the `retry` script succeeded after three failed tries.
 
+## install-package
+
+The `install-package` script can be used to cache missing packages and their dependencies.
+It works by creating a "temporary package cache" folder(if its not present), downloading packages(that are not present),  
```

Comment:
missing spaces before brackets I think:
=> It works by creating a "temporary package cache" folder (if its not present), downloading packages (that are not present),